test/spy-test.js

```diff
@@ -240,7 +240,26 @@
             }
         },
 
-        call: {
+        // test for regression introduced in #692. See #1046 for details
+        "localStorage support": {
+            requiresSupportFor: {
+                browser: typeof window !== "undefined",
+                localStorage: typeof window.localStorage !== "undefined"
+            },
+
+            "can spy on localStorage": function () {
+                var spy = sinon.spy(localStorage, "getItem");
+
+                localStorage.clear();
+                localStorage.setItem("foo", "fooval");
+                localStorage.getItem("foo");
+
+                assert(typeof localStorage.getItem.restore, "function" );
+                assert(spy.firstCall.returnValue === "fooval");
```

Comment:
Probably, right, but I am actually just trying out these changes, and it turned out that it (as assumed) breaks in Firefox (as stubbing localStorage cannot be done), so both the test and the heading is wrong. 

Do you have any idea how I can make a decent test that will check my changes? While this does revert the problems introduced in #692 it certainly has little to do with localStorage explicitly - that was just a side effect. But perhaps this is one area where unit tests add little value, as we are crossing the boundary over to browser internals, where browser specifics come into play. So perhaps I should remove the test?